deploy/examples/cluster-test.yaml

```diff
@@ -29,7 +29,7 @@ metadata:
 spec:
   dataDirHostPath: /var/lib/rook
   cephVersion:
-    image: quay.io/ceph/ceph:v17
+    image: quay.io/ceph/ceph:v18
```

Comment:
Were the canary tests passing? If they are still failing, we will need to change this back to v17 as well.

Reply:
I see the Canary tests passed in the previous run [here](https://github.com/rook/rook/actions/runs/5512055414) so it must be fine.